### ISSUE TITLE
Add link to user profile from sender's name

### ISSUE THREAD

**Issue by alya** [zulip/zulip-mobile#4866]
In the message feed, a user's avatar opens the profile menu, but the sender's name is currently not a link. We should make the name link to the profile menu as well, which is likely what the user expects, and is consistent with the web app UI.

**Comment by jay719** [comment]
@zulipbot claim

**Comment by SilentCruzer** [comment]
Can I work on this issue?

**Comment by jay719** [comment]
Yes please take it away from me, my computer has not been working for a
while

**Comment by chrisbobbe** [comment]
(Discussion has continued [on CZO](https://chat.zulip.org/#narrow/stream/48-mobile/topic/Edit.20WebView/near/1252673).)
